EntAppProject/JobBookWelcomeScreen.html

```diff
@@ -1,45 +1,28 @@
-<!DOCTYPE html>
-<html>
-<head>
-<title>Welcome to JobBook</title>
-<link rel="stylesheet" href="JobBookCSS.css">
-</head>
-<body>
- 
-<h1><span>Job</span>Book</h1>
- 
-<h2>Welcome to JobBook</h2>
-<p></p>
- 
-<form action="action_page.php" method="post">
-<div class="container">
-        <label for="username"><b>Username</b></label>
-        <input type="text" placeholder="Username:" name="username" required>
-        <br>
-        <br>
-        <label for="userpassword"><b>Password</b></label>
-        <input type="password" placeholder="Password" name="userpassword" required>
-        <br>
-        <form method="get" action="file:///C:/Users/salzm/OneDrive/Documents/School/Enterprise%20AppDev/JobBookUI/JBHomeScreen.html">

```

Comment:
Nesting forms within each other is not good practice to my understanding, and I am not even sure if it is possible. If I understand you intentions correctly, the first form pushes the content, and then the second form redirects to the next page. What you want to do instead is manage the redirection of the navigation based on the logic in the endpoint that the first form connects to. It will accomplish the same purpose (pushing data and redirection) without having to nest two forms together.

Reply:
Good catch... correct!